server/v2/testdata/config.toml

```diff
@@ -21,25 +21,13 @@ proxy_app = "tcp://127.0.0.1:26658"
 # A custom human readable name for this node
 moniker = "aurn-node"
 
-# Database backend: goleveldb | cleveldb | boltdb | rocksdb | badgerdb
```

Comment:
This file is generated, you need to modify that somewhere else first and run the server v2 test.